packages/gatsby-source-shopify/src/make-source-from-operation.ts

```diff
@@ -20,7 +20,7 @@ export function makeSourceFromOperation(
 ) {
   return async function sourceFromOperation(
     op: IShopifyBulkOperation,
-    isPriorityBuild = process.env.IS_PRODUCTION_BRANCH === `true`
+    isPriorityBuild = process.env.GATSBY_IS_PREVIEW !== `true`
```

Comment:
Wont we still have issues with PR build previews? Should we do `process.env.IS_PRODUCTION_BRANCH === 'true' && process.env.GATSBY_IS_PREVIEW !== 'true'`?

Reply:
Good catch, I always forget about PR builds! Pushed a change

Reply:
Just following up as this PR is getting a bit stale. Can it be merged after this update?